src/redux/reducers/process-resources.js

```diff
@@ -0,0 +1,97 @@
+// processResource() executes once per/resource, as paginated FHIR requests resolve.
```

Comment:
broke all of the following out from fhirReader, b/c none of these functions should be used by Components, and: they are used only once, to inject custom fields: